code/modules/client/preferences_savefile.dm

```diff
@@ -519,9 +523,11 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	features["spider_mandibles"]	= sanitize_inlist(features["spider_mandibles"], GLOB.spider_mandibles_list, "Plain")
 	features["moth_markings"] 	= sanitize_inlist(features["moth_markings"], GLOB.moth_markings_list, "None")
 	features["squid_face"]	= sanitize_inlist(features["squid_face"], GLOB.squid_face_list, "Squidward")
+	// Void edit begin
```

Comment:
same here, i think void edit mentions should be letting us know when we're gonna keep something through next tg update, which tgui prefs menu kills